app/styles/ui/_blank-slate.scss

```diff
@@ -1,17 +1,17 @@
 #blank-slate {
   display: flex;
-  justify-content: center;
 
   .header {
```

Comment:
I think we can remove this now, right? I'm gonna go ahead and merge anyway since I want this in asap but perhaps you could open another PR to clean this rule up?